package-res/resources/web/pentaho/visual/base/events/RejectedUpdate.js

```diff
@@ -19,18 +19,48 @@ define([
 ], function(Event, mixinError) {
   "use strict";
 
-  return Event.extend("pentaho.visual.base.events.RejectedUpdate", {
+  /**
+   * @name RejectedUpdate
+   * @memberOf pentaho.visual.base.events
+   * @class
+   * @extends pentaho.lang.Event
+   * @mixes pentaho.lang.mixins.mixinError
+   *
+   * @classDesc This event is emitted when any rejection occurs while updating
+   * the view with {@link pentaho.visual.base.Model#update|Model#update}.
+   *
+   * A rejection can be one of the following:
+   *  - the event {@link pentaho.type.events.WillUpdate|"will:update"} was canceled
+   *  - an error occurred while validating the view
+   *  - an error occurred while updating the view
+   *
+   * @constructor
+   * @description Creates a `RejectedUpdate` event.
+   *
+   * @param {!pentaho.visual.base.Model} source - The model object which is emitting the event.
```

Comment:
@param {!pentaho.visual.base.Model} source - The model object that is emitting the event.